client/src/components/Panels/Common/ToolAdvancedSearch.vue

```diff
@@ -0,0 +1,219 @@
+        :show-help="showHelp === 'true'">
+        <section>
+            <div class="mb-2">
+                <span class="row">
+                    <span class="col">
+                        <h4 class="d-inline-block align-middle">Advanced Tool Search Results</h4>
+                        <b-button v-if="hasFilters" class="float-right" size="sm" @click="onClear">
+                            <icon icon="redo" />
+                            <span>{{ "Clear Search" | localize }}</span>
+                        </b-button>
+                    </span>
+                </span>
+                <span v-if="itemsLoaded.length !== 0 && hasFilters" class="row">
+                    <span class="col"> (found {{ itemsLoaded.length }} of {{ totalItems }} tools) </span>
+                </span>
+            </div>
+            <div>
+                <b-alert v-if="loading" class="m-2" variant="info" show>
+                    <LoadingSpan message="Loading Advanced Search Results" />
+                </b-alert>
+                <b-alert v-else-if="!hasFilters" class="m-2" variant="secondary" show>
+                    There are no valid filters applied. Please add filters to the menu in the Tool Panel.
+                </b-alert>
+                <b-alert v-else-if="!itemsLoaded || itemsLoaded.length == 0" class="m-2" variant="danger" show>
+                    No tools found for selected filter(s): {{ filterSettings }}
+                </b-alert>
+                <div v-else>
+                    <b-card v-for="(tool, key) in itemsLoaded" :key="key" no-body>
+                        <b-card-header
+                            v-b-toggle="'accordion-' + key"
+                            :disabled="!hasHelp(tool.help)"
+                            :role="hasHelp(tool.help) ? 'button' : ''">
+                            <span class="pl-2 d-flex justify-content-between">
+                                <span class="row">
+                                    <a
+                                        :href="tool.target === 'galaxy_main' ? 'javascript:void(0)' : tool.link"
+                                        @click.stop="onOpen(tool)">
+                                        <b>{{ tool.name }}</b> {{ tool.description }}
+                                    </a>
+                                    <b-badge class="ml-1">{{ tool.panel_section_name }}</b-badge>
+                                    <b-badge
+                                        v-if="tool.is_workflow_compatible"
+                                        v-b-tooltip.hover
+                                        class="ml-1"
+                                        variant="success"
+                                        title="Can use this tool in Workflows"
+                                        >Workflow</b-badge
+                                    >
+                                    <b-badge v-if="tool.hidden" class="ml-1" variant="danger">Hidden</b-badge>
```

Comment:
Somehow, the badges have different heights:

<img width="581" alt="image" src="https://user-images.githubusercontent.com/2105447/186536625-8cc857db-b29b-44c3-9161-aa3a762d8b3b.png">

Reply:
The muted "Click here to expand tool info" is difficult to see, maybe try an expand icon or just write "...more" with higher contrast?

Reply:
Getting rid of that in an incoming push